app/models/book.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Book < ApplicationRecord
+  has_many :reviews, dependent: :destroy
+  validates :name, presence: true
```

Comment:
better to add empty line to separate association zone and validation zone to read easier